python/tank/platform/qt/tankqdialog.py

```diff
@@ -13,7 +13,7 @@
 
 """
 
-from . import QtCore, QtGui
+from . import QtCore, QtGui, QtWidgets
```

Comment:
Hum we should import qt6 here right?

Should we move all those modules to the qt6 folder instead?